torch/_inductor/cpp_builder.py

```diff
@@ -366,14 +366,26 @@ def _get_cpp_std_cflag(std_num: str = "c++17") -> List[str]:
         return [f"std={std_num}"]
 
 
-def _get_linux_cpp_cflags(cpp_compiler: str) -> List[str]:
-    if not _IS_WINDOWS:
+def _get_os_related_cpp_cflags(cpp_compiler: str) -> List[str]:
+    if _IS_WINDOWS:
+        cflags = [
+            "wd4819",
+            "wd4251",
+            "wd4244",
+            "wd4267",
+            "wd4275",
+            "wd4018",
+            "wd4190",
+            "wd4624",
+            "wd4067",
+            "wd4068",
```

Comment:
Perhaps it makes sense to summarize what warnings we want to disable here?

Reply:
@jgong5 I will add comments for detailed in further PR.